src/value.rs

```diff
@@ -998,6 +1023,24 @@ impl<'a> From<&'a [u8]> for Value {
 	}
 }
 
+/// Value from time/date.
+impl From<std::time::SystemTime> for Value {
+	/// See the [VALUE_TIME::T_DATE].
+	fn from(val: std::time::SystemTime) -> Self {
+		let mut me = Value::new();
+		if let Ok(epoch) = val.duration_since(std::time::UNIX_EPOCH) {
+			// in 100ns from epoch
+			let ns = epoch.as_nanos() / 100;
+
+			// does not fail because does not check the input
+			// TODO: is SystemTime UTC or not?
```

Comment:
It is, according to [`SystemTime`: underlaying system calls](https://doc.rust-lang.org/nightly/std/time/struct.SystemTime.html#underlying-system-calls) and [`GetSystemTime`](https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getsystemtimeasfiletime).

Reply:
But in Rust itself it's unspecified - see the other platforms' docs.